packages/server/src/models/user.entity.ts

```diff
@@ -13,4 +13,7 @@ export default class User {
 
   @Column()
   public password: string;
+
+  @Column()
+  public status: string;
```

Comment:
Mb should clarify what status it is?

Reply:
you can check UserStatuses enum in first commit